pallets/claims/Cargo.toml

```diff
@@ -0,0 +1,64 @@
+[package]
+authors = ['GalacticCouncil']
+description = 'HydraDX Claims Module'
+edition = '2018'
+homepage = 'https://github.com/galacticcouncil/hack.hydradx-node'
+license = 'Unlicense'
+name = 'pallet-claims'
+repository = 'https://github.com/galacticcouncil/hack.hydradx-node'
+version = '2.0.0'
+
+[package.metadata.docs.rs]
+targets = ['x86_64-unknown-linux-gnu']
+
+# alias "parity-scale-code" to "codec"
+[dependencies.codec]
+default-features = false
+features = ['derive']
+package = 'parity-scale-codec'
+version = '1.3.4'
+
+[dependencies]
+# Substrate dependencies
+frame-support = { default-features = false, version = '2.0.0' }
+frame-system = { default-features = false, version = '2.0.0' }
+frame-benchmarking = { version = "2.0.0", default-features = false, optional = true }
+rustc-hex = { version = '2.1.0', default-features = false }
+serde = { optional = true, version = '1.0.101', features = ['derive'] }
+sp-io = { default-features = false, version = '2.0.0' }
+sp-runtime = { default-features = false, version = '2.0.0' }
+sp-std = { default-features = false, version = '2.0.0' }
+hex = {default-features = false, version = "0.4.2"}
+hex-literal = '0.3.1'
+lazy_static = {features = ['spin_no_std'], version = "1.4.0"}
+
+# Local dependencies
+primitives = {path = '../../primitives', default-features = false, version = '2.0.0'}
+
+# ORML dependencies
+orml-traits = { default-features = false, version = '0.3.3-dev' }
+orml-tokens = { default-features = false, version = '0.3.3-dev' }
+orml-utilities = { default-features = false, version = '0.3.2' }
```

Comment:
We should use the same version - 0.3.3-dev. I believe the reason why it is not found when you try to use it - is that it is not patched in the Cargo file in the top level repo dir. If you add the orml-utilies to patch section of that cargo file with the same revision as the other orml crates - you will be able to use 0.3.3-dev version.